.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,98 @@
+name: 'Bug report'
+title: "[BUG] - YOUR_ISSUE_TITLE_HERE_REPLACE_ME"
+description: Create a report to help us improve
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue :pray:.
+
+        This issue tracker is for reporting bugs found in [NextUI github repository](https://github.com/nextui-org/nextui/)
+        If you have a question about how to achieve something and are struggling, please post a question
+        inside of either of the following places:
+          - NextUI's [Discussion's tab](https://github.com/nextui-org/nextui/discussions)
+          - NextUI's [Discord channel](https://foambubble.github.io/join-discord/g)
```

Comment:
Hey @cliffordfajardo can you please update Discord Link , I did mentioned this in last review report looks like you forget to fix it 😄

Reply:
WHoops - apologies - the code has been merged but a new PR can be created

Reply:
Do you want to open a new PR ?

Reply:
Hey @cliffordfajardo @Dhananjay-JSR sorry I didn't notice that, fixed it now 👍🏻